### ISSUE TITLE
Code blocks not surrounded by <pre> tags

### ISSUE THREAD

**Issue by atodorov** [Python-Markdown/markdown#701]
```
>>> import markdown
>>> markdown.markdown('```this is a code block```')
'<p><code>this is a code block</code></p>'
```

This is inconsistent with behavior I've seen elsewhere on the web and also inconsistent with a JavaScript editor that we use (Simple MDE).

**Comment by waylan** [comment]
It is completely consistent with the [rules][1] (and the reference implementation), which permit inline code **spans** to use any number of backticks. In fact, as [Babelmark][3] demonstrates, nearly every implementation will render a single line that way.

That said, I assume you intended to suggest that fenced code blocks are not supported. First, your example is not a fenced code block as there are no line breaks. Second, fenced code blocks are not part of standard Markdown. That said, we do provide an [extension][2] which adds support for them.

In other words, what you probably want is this:

```python
>>> markdown.markdown('```\nthis is a code block\n```', extensions=['fenced_code'])
'<pre><code>this is a code block\n</code></pre>'
```

[1]: http://daringfireball.net/projects/markdown/syntax
[2]: https://python-markdown.github.io/extensions/fenced_code_blocks/
[3]: https://johnmacfarlane.net/babelmark2/?normalize=1&text=%60%60%60this+is+a+code+block%60%60%60
